### PR TITLE
rum collection test

### PR DESCRIPTION
**Pull request by davidnuescheler**
https://rum-collection-test--spark-website--adobe.hlx3.page/express/learn/blog?cwv=on

### CODE REVIEW

File: express/scripts/martech.js [hunk 1]
```diff
@@ -406,6 +407,19 @@ loadScript('https://www.adobe.com/marketingtech/main.min.js', () => {
 
   decorateAnalyticsEvents();
 
+  const { weight, id, random } = window.hlx.rum;
+  const generation = 'instrumentation-test-martech';
+  window.hlx.rum.generation = generation;
```
Comment (kptdobe) [inline express/scripts/martech.js:412]: Why do you need to set `window.hlx.rum.generation` ? You never re-use it.

File: express/scripts/martech.js [hunk 2]
```diff
@@ -516,40 +530,28 @@ loadScript('https://www.adobe.com/etc.clientlibs/globalnav/clientlibs/base/feds.
...
   rum.referer = window.location.href;
-  store(rum);
+  if (generation) rum.generation = generation;
+  sendRUMData(rum, weight);
 }
 
-if (Math.random() * weight < 1) {
+const { weight, id, random } = window.hlx.rum;
+const generation = 'instrumentation-test';
+window.hlx.rum.generation = generation;
```
Comment (kptdobe) [inline express/scripts/martech.js:545]: Same here.

**Review by kptdobe — APPROVED** [review]

File: express/scripts/martech.js (hunk 2 above)
Comment (davidnuescheler) [reply express/scripts/martech.js:545]: this one is used in the CWV submissions...

File: express/scripts/martech.js (hunk 1 above)
Comment (davidnuescheler) [reply express/scripts/martech.js:412]: great catch, this one was actually wrong.

**Review by trieloff — APPROVED** [review]

**Pull request merged**